src/System.Net.Http/src/System/Net/Http/HttpRequestException.cs

```diff
@@ -25,5 +28,16 @@ public HttpRequestException(string message, Exception inner)
                 HResult = inner.HResult;
             }
         }
+
+        // This constructor is used internally to indicate that a request was not successfully sent due to an IOException,
+        // and the exception occurred early enough so that the request may be retried on another connection.
+        // occurred before the request
```

Comment:
Nit: this last part of the comment looks wrong

Reply:
fixed